CMakeLists.txt

```diff
@@ -377,50 +402,48 @@ add_library(cradle_rpclib_client STATIC)
 target_sources(cradle_rpclib_client PRIVATE
     ${srcs_rpclib_client}
     ${deploy_dir_file})
-target_compile_definitions(
-    cradle_rpclib_client PUBLIC ${CONAN_DEFINES} ${CONAN_COMPILE_DEFINITIONS})
-target_include_directories(cradle_rpclib_client PRIVATE
+record_public_include_directory(cradle_rpclib_client
```

Comment:
There shouldn't be any `*.hpp` in `cradle_rpclib_client`.

Reply:
This might've also been caused by the above issue, but in any case, you're correct. It should be fixed now.